### PR TITLE
feat: add lumin blog article

### PR DESCRIPTION
**Pull request by RinorG**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

File: src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+teaser: >
+  Ory’s flagship zero trust network  “Ory Cloud”  is a global, scalable and secure platform for managing identities and its distribution across various internet services.
+tags:
+  - Guide
+  - Access Control
+  - Guest Author
+  - Security & Compliance
+---
+
+# How Ory Helps Lumin Protect Their Users’ Data
```
Comment (jfcurran) [inline src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx:31]:
```suggestion
## How Ory Helps Lumin Protect Their Users’ Data
```

File: src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+"We are continually thinking about how to better protect our data, using techniques like encryption and role-based access control. Our first move was to implement a multi-layered strategy where data is protected by many strong layers of security technology. Many of the targeted companies are doing the same." Max Ferguson, CEO Lumin PDF, commented in a blogpost on [LuminPDF data breach](https://www.luminpdf.com/blog/is-lumin-pdf-editor-safe/) on 1st of April, 2019.
+
+Since then, [Lumin](https://www.luminpdf.com/) has integrated multiple security measures to protect its users’ privacy and the company servers. With a customer base of over 65 million and counting, it’s no surprise that this New Zealand-based software company would leave any stone unturned to scale up its security measures.
+
+## A Brief Intro to Lumin PDF
```
Comment (jfcurran) [inline src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx:37]:
```suggestion
### A Brief Intro to Lumin PDF
```

File: src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+Since its establishment in 2014, the cloud-based PDF collaboration and editing platform offers full integration with popular tools like Google Drive and Dropbox. Customers use its extensive toolkit to create, share and edit the documents they use at work and in their personal lives. Pairing up with giants like Blockchain, Netflix and Uber, Lumin shot up for [growth by 600 percent](https://www.rnz.co.nz/news/business/456098/christchurch-based-software-firm-lumin-pdf-has-60m-users-worldwide-and-growing), making it one of the fastest growing tech companies in 2021. 
+While the number of global user profiles keep growing, Lumin is harboring sensitive data on a daily basis. The software caters to a wide audience - from educators to business administrators and beyond. Lumin needs to add more stringent authentication, authorization and access control security. 
+This is where we come in.  
+
+## How Ory Helps Lumin
```
Comment (jfcurran) [inline src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx:42]:
```suggestion
### How Ory Helps Lumin
```

File: src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+Ory’s flagship zero trust network  “Ory Cloud”  is a global, scalable and secure platform for managing identities and its distribution across various internet services. Ory is a fast-growing startup offering a suite of services that make it easier for developers and businesses to create secure and dependable web applications and services. 
+Offering capabilities such as single sign-on, self-service password resets and custom branding, Ory is an end-to-end solution for identity management, authorization, authentication and access control on any web platform.
+
+Using Ory’s system ensures users’ login credentials are secure through two-factor authentication as an additional layer of protection. It also means that Lumin is able to provide enterprise-grade login and security, which is crucial for some of its business and enterprise customers. Currently, Lumin uses Ory Kratos, Ory Hydra, and Ory Keto.
+
+### Ory Kratos
```
Comment (jfcurran) [inline src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx:48]:
```suggestion
#### Ory Kratos
```

File: src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+Lumin uses [Ory Kratos](https://www.ory.sh/kratos/) to manage sign-in and session management for Lumin customers. Ory Kratos provides tools for user management and authentication. It also allows customers to access data on other platforms through its Single Sign-On (SSO) capability, as well as protects the users of a web application from data breaches and phishing attempts through Self-Service Password Reset (SSPR). 
+
+### Ory Hydra
```
Comment (jfcurran) [inline src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx:51]:
```suggestion
#### Ory Hydra
```

File: src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+Lumin also uses [Ory Hydra](https://www.ory.sh/hydra/) for authenticating requests to their open API. Ory Hydra manages all of the security tokens and servers to verify users who have logged into an application. Once verified, it creates an authorization token that’s used to access restricted content on its partner applications. 
+
+### Ory Keto
```
Comment (jfcurran) [inline src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx:54]:
```suggestion
#### Ory Keto
```

File: src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+Another project on the horizon for Lumin powered by Ory is the template library. This extensive library of readily available templates will use Ory’s search function to help sort through multiple documents to find the right fit for them. Ory Keto [Ory Keto](https://www.ory.sh/keto) access control will be used to manage user accessibility to specific templates. The project ties in with Lumin’s wider vision; to ensure users can find, sign and share a document within 30 seconds.
+
+## The Benefits of Working Together
```
Comment (jfcurran) [inline src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx:57]:
```suggestion
### The Benefits of Working Together
```

File: src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+### Flexibility
```
Comment (jfcurran) [inline src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx:59]:
```suggestion
#### Flexibility
```

File: src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+Lumin’s philosophy centers around accessibility, with founder Max Ferguson wanting to create a platform accessible to everyone no matter where they are in the world. It offers a free version, giving users access to many of its popular features. Similarly, Ory’s open-source technology is readily accessible, meaning Lumin can offer the same to their customers. 
+The Lumin team is currently working on several new open-source projects to make it easier for developers to work with PDF files in their own applications. These projects will leverage the Lumin API and allow the developers to integrate with Lumin technology. 
+Unlike some of their competitors, Ory’s open-source nature gives Lumin’s developers the flexibility to review code and suggest changes, giving input into the functionality to ensure it best fits their use-case.
+
+### Usability
```
Comment (jfcurran) [inline src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx:64]:
```suggestion
#### Usability
```

File: src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+The Lumin team has over 50 dedicated developers continuously working on growing the platform with a multitude of software partners, so flexibility and ease of use are paramount. Lumin’s System Architect based in Vietnam, Hoan Tran, says working with Ory “opens up a wide variety of concepts that once explored, really show you how flexible and adaptable the software can be”.
+
+### Affordability and Scalability
```
Comment (jfcurran) [inline src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx:67]:
```suggestion
#### Affordability and Scalability
```

File: src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+
+Since then, [Lumin](https://www.luminpdf.com/) has integrated multiple security measures to protect its users’ privacy and the company servers. With a customer base of over 65 million and counting, it’s no surprise that this New Zealand-based software company would leave any stone unturned to scale up its security measures.
+
+## A Brief Intro to Lumin PDF
+Since its establishment in 2014, the cloud-based PDF collaboration and editing platform offers full integration with popular tools like Google Drive and Dropbox. Customers use its extensive toolkit to create, share and edit the documents they use at work and in their personal lives. Pairing up with giants like Blockchain, Netflix and Uber, Lumin shot up for [growth by 600 percent](https://www.rnz.co.nz/news/business/456098/christchurch-based-software-firm-lumin-pdf-has-60m-users-worldwide-and-growing), making it one of the fastest growing tech companies in 2021.
```
Comment (jfcurran) [inline src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx:38]: Blockchain?

**Review by jfcurran — CHANGES_REQUESTED** [review]

File: src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx (hunk 11 above)
Comment (jfcurran) [reply src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx:38]: I am pretty sure this is Blockchain.com. Simply mentioning Blockchain sounds very suspect. Please clarify @RinorG @BlueAcidFrog

Comment (RinorG) [reply src/markdown/blog/how-ory-helps-lumin-protect-their-users.mdx]: changed it to blockchain.com

**Review by jfcurran — APPROVED** [review]

**Pull request merged**